metadata.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "puppet-pulpcore",
+  "version": "0.1.0",
+  "author": "William Clark",
+  "summary": "",
+  "license": "GPLv3",
+  "source": "",
+  "dependencies": [
+    {
+      "name": "puppet/redis",
+      "version_requirement": ">= 4.2.0 < 5.0.0"
+    },
+    {
+      "name": "puppetlabs/postgresql",
+      "version_requirement": ">= 4.2.0 < 7.0.0"
+    }
+  ],
+  "operatingsystem_support": [
+    {
+      "operatingsystem": "CentOS",
+      "operatingsystemrelease": [
+        "7"
+      ]
+    },
+    {
```

Comment:
I'd not be too pro-active with listing support. I'd just list Red Hat 7 and CentOS 7.